src/GraphQL/Types/TypeInformation.cs

```diff
@@ -0,0 +1,258 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace GraphQL.Types
+{
+    /// <summary>
+    /// Contains type and nullability information for a method return type or argument type.
```

Comment:
> for a method return type or argument type

I see only usages with `PropertyInfo`.

Reply:
Prep for the next pull request, which is mostly done now. I tried to eliminate references in here but must have missed this one.

Reply:
Rewrote this comment to read

> Parses a `System.Type` along with nullability information into its constituent parts
> in preparation for creating a graph type that represents such a type.

Reply:
OK